planner/core/plan_cache_utils.go

```diff
@@ -325,12 +345,52 @@ type PlanCacheValue struct {
 	OutPutNames       []*types.FieldName
 	TblInfo2UnionScan map[*model.TableInfo]bool
 	ParamTypes        FieldSlice
+	memoryUsage       int64
 }
 
 func (v *PlanCacheValue) varTypesUnchanged(txtVarTps []*types.FieldType) bool {
 	return v.ParamTypes.CheckTypesCompatibility4PC(txtVarTps)
 }
 
+// unKnownMemoryUsage represent the memory usage of uncounted structure, maybe need implement later
+// 7 has no special meaning
+const unKnownMemoryUsage = 7
```

Comment:
Seems too small. Actually, I have tested it before and got an empirical value `100KB`, you can see it in our doc https://docs.pingcap.com/tidb/dev/sql-prepared-plan-cache#memory-management-of-prepared-plan-cache